services/calculate_analytics/analyses/calculate_feed_topic_models_2025_08_18/train/run_training_local.sh

```
@@ -0,0 +1,40 @@
+FORCE_FALLBACK=""
+
+echo "🚀 Starting Local Topic Model Training"
+echo "======================================"
+echo "Mode: $MODE"
+echo "Sample per day: $SAMPLE_PER_DAY"
+echo "Output dir: ${OUTPUT_DIR:-'./trained_models/local (default)'}"
+echo "Force fallback: ${FORCE_FALLBACK:-'no'}"
+echo ""
+
+# Change to script directory
+cd "$SCRIPT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)